openapi/components/schemas/Transactions/Transaction.yaml

```
@@ -239,6 +239,9 @@ allOf:
         readOnly: true
         allOf:
           - $ref: ../CurrencyCode.yaml
+      organizationId:
+        allOf:
+          - $ref: ../OrganizationId.yaml
       settlementTime:
         type: string
         description: The time that the transaction was settled by the banking instuition.
```

nitpick: suggestion to add more detail and remove to avoid starting with "the"
```suggestion
        description: Date and time at which the transaction was settled by the banking institution.
```